### PR TITLE
Make a Lightning payment. Fix LNURL pay section with lncli command 

### PR DESCRIPTION
**Pull request by alanStocco**
Section "lightning->lightning-client ->Make a Lightning payment"
it's write to pay in  tippin.me.
But in these way you get a LNURL that not works in LND.

#### What

In the guide section, to try first LN invoice payment,  the link to  tippin me  gives an  LNURL .

### Why

LNURL not works in LND right now.

#### How

Write a better explanation and a cli command to pay the Stadicus node

#### Scope

- [ ] significant change to core configuration
- [x] independent bonus guide
- [ ] simple bug fix

### CODE REVIEW

**Review by VajraOfIndra — COMMENTED** [review]
I propose that we remove the LNURL things altogether as it's a bit confusing IMHO.

Instead, I think having a short introductory sentence that explains keysend payments and then the lncli keysend command would be enough and clearer.

E.g. something along the lines of:

* **Make a Lightning payment**. By default, these work with invoices, so when you buy something or want to send money, you need to get an invoice first. However, you can also pay without requesting an invoice as long the receiving node supports the keysend or amp feature!

To try, why not send me a single satoshi! You simply need to input [my node pukey]( ([https://amboss.space/node/02acd93e3352fd59066ca3f23e8865de1926301e8be03c6a52f0f7e43533fe9888){:target="_blank"})](https://amboss.space/node/02acd93e3352fd59066ca3f23e8865de1926301e8be03c6a52f0f7e43533fe9888)%7b:target=%22_blank%22%7d)), the amount in satoshis and add the `–keysend` flag.

    ```sh
    * lncli sendpayment --dest 02acd93e3352fd59066ca3f23e8865de1926301e8be03c6a52f0f7e43533fe9888 --amt 1 --keysend
    ```

**Comment by twofaktor** [comment]
Comments made in code!!

File: guide/lightning/lightning-client.md [hunk 1]
```diff
@@ -519,15 +519,24 @@ Just grab the whole URI above the big QR code and use it as follows (we will use
...
+  
   To try, why not send me a single satoshi to view my Twitter profile?
 
   * Click on this Tippin.me link: <https://tippin.me/@Stadicus3000>
   * Click on "Copy request" to copy the invoice data
   * Pay me 1 satoshi (~ $0.0001) 🤑
 
     ```sh
-    * lncli payinvoice lnbc10n1pw......................gsj59
+    * lncli payinvoice lnurl..........
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:532]: This doesn't work in `lncli`. Delete the command, the method was already specified in 522/523 line!

Captura2

File: guide/lightning/lightning-client.md [hunk 2]
```diff
@@ -519,15 +519,24 @@ Just grab the whole URI above the big QR code and use it as follows (we will use
   ```
 
 * **Make a Lightning payment**. These work with invoices, so everytime you buy something or want to send money, you need to get an invoice first.
+Or you can pay directly another node.
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:522]: Delete this line. This was already specified in line 534

File: guide/lightning/lightning-client.md [hunk 3]
```diff
@@ -519,15 +519,24 @@ Just grab the whole URI above the big QR code and use it as follows (we will use
...
 
   * Click on this Tippin.me link: <https://tippin.me/@Stadicus3000>
   * Click on "Copy request" to copy the invoice data
   * Pay me 1 satoshi (~ $0.0001) 🤑
 
     ```sh
-    * lncli payinvoice lnbc10n1pw......................gsj59
+    * lncli payinvoice lnurl..........
     ```
+  Or you can do it using lncli , [`Stadicus node`](https://amboss.space/node/02acd93e3352fd59066ca3f23e8865de1926301e8be03c6a52f0f7e43533fe9888){:target="_blank"}
```
Comment (twofaktor) [inline guide/lightning/lightning-client.md:534]: Change to: Or you can do it using `keysend` tag of the next 'lncli' command paying directly to [`Stadicus node`](https://amboss.space/node/02acd93e3352fd59066ca3f23e8865de1926301e8be03c6a52f0f7e43533fe9888){:target="_blank"}  pubkey.

**Review by twofaktor — COMMENTED** [review]
Suggestions in the comments of the code

File: guide/lightning/lightning-client.md [hunk 4]
```diff
@@ -518,20 +518,9 @@ Just grab the whole URI above the big QR code and use it as follows (we will use
   $ lncli listchannels
   ```
 
-* **Make a Lightning payment**. These work with invoices, so everytime you buy something or want to send money, you need to get an invoice first.
-Or you can pay directly another node.
-LND not support LNURL so you can pay an invoice using a mobile wallet that support LNURL
+* **Make a Lightning payment**. TBy default, these work with invoices, so when you buy something or want to send money, you need to get an invoice first. However, you can also pay without requesting an invoice as long the receiving node supports the keysend or amp feature!
```
Comment (VajraOfIndra) [inline guide/lightning/lightning-client.md:521]:
```suggestion
* **Make a Lightning payment**. By default, these work with invoices, so when you buy something or want to send money, you need to get an invoice first. However, you can also pay without requesting an invoice as long the receiving node supports the keysend or amp feature!
```

Just a tiny nit and le'ts merge it!

**Review by VajraOfIndra — APPROVED** [review]

**Comment by Stadicus** [comment]
Yes, the Tippin.me link has been broken for a long time after it was taken over by some other operator and they moved to LNURL. I don't have a good solution for sending a single sat at the moment.

My nodes are not a good target, they are not meant to run forever and I keep setting up new ones... Having a shared tipping jar for the RaspiBolt project doesn't seem to be worth the trouble, as we don't have any real expenses. 

Maybe we could add instructions how to donate a few sats with a proper invoice? For example the OpenSats page for Zeus?

https://opensats.org/projects/zeus

**Comment by VajraOfIndra** [comment]
> I don't have a good solution for sending a single sat at the moment.

A keysend (or amp) payment to your node using lncli works fine (but i see your second point about not keeping same node for long so yeah)

> My nodes are not a good target, they are not meant to run forever and I keep setting up new ones... 
Having a shared tipping jar for the RaspiBolt project doesn't seem to be worth the trouble, as we don't have any real expenses.

Only the domain name that you have to pay right? Sats could go to that if you want...

> Maybe we could add instructions how to donate a few sats with a proper invoice? For example the OpenSats page for Zeus?
> https://opensats.org/projects/zeus

Love the idea of the RaspiBolt contributing to the developpers that make the apps we use in the guide... What about trying to split between all the different projects we use in the main guide and that accept donation? would be fairer but maybe too hard to implement? just thinking out loud, happy to start with Zeus for now.

File: guide/lightning/lightning-client.md (hunk 4 above)
Comment (VajraOfIndra) [reply guide/lightning/lightning-client.md:521]: @alanStocco could you commit this small typo fix? That would avoid us having to open a new PR just to fix it later. Thanks

Comment (alanStocco) [reply guide/lightning/lightning-client.md]: @VajraOfIndra  thanks. Done.

**Comment by VajraOfIndra** [comment]
I propose we merge this fix rn as people keep enocuntering the lnurl bug.

If a donation to Zeus or other FOSS Bitcoin projects is desired, this could be done by a subsequent PR.

**Pull request merged**